apps/prairielearn/python/prairielearn.py

```diff
@@ -1173,7 +1195,9 @@ def string_fraction_to_number(a_sub, allow_fractions=True, allow_complex=True):
     return (value, data)
 
 
-def string_to_2darray(s, allow_complex=True):
+def string_to_2darray(
+    s: str, allow_complex=True
```

Comment:
Boolean?

Reply:
Fixed in 5027838266fba11cc95373b6954f1c928a030c78.